LogicLayer/GameEventParticipations/GameEventParticipationService.cs

```diff
@@ -63,6 +64,29 @@ public async Task SendGameEventInvitationAsync(SendGameEventInvitationDto gameEv
             }
         }
 
+        public async Task SendGameEventInvitationAsync(IEnumerable<SendGameEventInvitationDto> gameEventInvitationDtos)
+        {
+            using (var transaction = _repository.BeginTransaction())
+            {
+                foreach (var gameEventInvitationDto in gameEventInvitationDtos)
+                {
+                    try
+                    {
+                        await SendGameEventInvitationAsync(gameEventInvitationDto);
+                    }
+                    catch (ArgumentOutOfRangeException exception)
+                    {
+                        throw exception;
```

Comment:
If you're rethrowing caught exceptions, use `throw`, not `throw exception`.
```csharp
catch (ArgumentOutOfRangeException exception)
{
    throw;
}
```
Throwing this way preserves the stack-trace. The way it's now, the stack trace would begin at this point in code and we'd lose all the info from the DataLayer.

Reply:
Marking as done.